Architect-Coders/MU8
# Issue #37: Data Binding on Characters Activity

**Data Binding on Characters Activity** #37

-  Delete charactersUiModel because implemented databinding.
- Create diferent bindingAdapter for use in xml.
- Create Event for use navigation.

**Testing Done:**

- Nexus 6P

**How to test:**

- Run the application and click onCharacter Category
- You can see the loading while the characters have been loaded
- Then you can see the characters in a scrollable list

**Signed-off-by**:

- Javier Cardona <javier.mcardona@gmail.com>
